test/contextstore.test.js

```diff
@@ -394,12 +394,11 @@ describe('Koa Session External Context Store', () => {
 
         request(server)
```

Comment:
if we remove `done`, we should `return request` to ensure this promise resolved.

Reply:
Is it correct like this? (I'm not familiar with this mode of usage of mocha).